ui/page/root/main_page.go

```diff
@@ -122,21 +111,21 @@ func (mp *MainPage) OnNavigatedTo() {
 	}
 
 	if mp.CurrentPage() == nil {
-		mp.Display(info.NewInfoPage(mp.Load)) // TODO: Should pagestack have a start page?
+		mp.Display(info.NewInfoPage(mp.Load)) // TODO: Should pagestack have a start page? YES!
```

Comment:
@ukane-philemon would you like to work on this?

Reply:
Yes. I have just 2 PRs under review.